logbook/oplog/log.go

```diff
@@ -75,6 +75,12 @@ type Logstore interface {
 	// use Logstore.Children or Logstore.Descendants to populate missing children
 	HeadRef(ctx context.Context, names ...string) (*Log, error)
 
+	// RetrieveRef returns the log that matches the username and dataset name
+	RetrieveRef(username, dsName string) (*Log, error)
+
+	// RetrieveByInitID returns the log that matches the initID
+	RetrieveByInitID(initID string) (*Log, error)
```

Comment:
This is equivalent to `Logstore.Log(ctx, initID)`, please use that instead.

Given you couldn't find this method easily, it's clear to me the `Log` method on the `Logstore` interface is poorly named. Should we rename to `Logstore.Get`?

Reply:
You're right, totally missed the `Log` method. Will rename it to `Get`.